pkg/oc/cli/newbuild/newbuild.go

```diff
@@ -224,7 +225,7 @@ func (o *BuildOptions) RunNewBuild() error {
 	return nil
 }
 
-func transformBuildError(err error, baseName, commandName, commandPath string, groups errorGroups, config *newcmd.AppConfig) {
+func transformBuildError(err error, baseName, commandName, commandPath string, groups ocnewapp.ErrorGroups, config *newcmd.AppConfig) {
```

Comment:
should ErrorGroups go into util?

Reply:
> should ErrorGroups go into util?

They appear to be a peculiarity of newapp.